pkg/signals/signal_posix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
 // +build !windows
```

Comment:
Is it necessary to keep both syntaxes?

Reply:
Not a necessity but the reference codebase which i referred has done the same: 

https://github.com/kubernetes/sample-controller/blob/master/pkg/signals/signal_posix.go